### PR TITLE
Updated visualize.tsx with the latest tools.jsp content

### PR DESCRIPTION
**Pull request by onursumer**
# What? Why?
Fix  https://github.com/cBioPortal/cbioportal/issues/4904

visualize-tools

# Checks
- [ ] Is this PR adding logic based on one or more **clinical** attributes? If yes, please make sure validation for this attribute is also present in the data validation / data loading layers (in backend repo) and documented in [File-Formats Clinical data section](https://github.com/cBioPortal/cbioportal/blob/master/docs/File-Formats.md#clinical-data)!
- [ ] Follows [7 rules of great commit messages](http://chris.beams.io/posts/git-commit/). For most PRs a single commit should suffice, in some cases multiple topical commits can be useful. During review it is ok to see tiny commits (e.g. Fix reviewer comments), but right before the code gets merged to master or rc branch, any such commits should be squashed since they are useless to the other developers. Definitely avoid [merge commits, use rebase instead.](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [ ] Follows the [Airbnb React/JSX Style guide](https://github.com/airbnb/javascript/tree/master/react).
- [ ] Make sure your commit messages end with a Signed-off-by string (this line
  can be automatically added by git if you run the `git-commit` command with
  the `-s` option)

### CODE REVIEW

File: end-to-end-tests/specs/mutationMapperTool.spec.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ describe('Mutation Mapper Tool', function() {
 
     describe('example genomic changes input', ()=>{
         beforeEach(()=>{
-            var url = `${CBIOPORTAL_URL}/mutation_mapper.jsp`;
+            var url = `${CBIOPORTAL_xURL}/s/mutation_mapper`;
```
Comment (onursumer) [inline end-to-end-tests/specs/mutationMapperTool.spec.js]: @alisman is this the right way to fix this route?

File: src/routes.jsx [hunk 2]
```diff
@@ -81,7 +81,7 @@ export const makeRoutes = (routing) => {
                 <Route path="/newstudy" onEnter={()=>{$(document).scrollTop(0)}} getComponent={lazyLoadComponent(StudyViewPage)} />
                 <Route path="/study" component={getBlankPage()} />
 
-                <Route path="/mutation_mapper" getComponent={lazyLoadComponent(MutationMapperTool)} />
+                <Route path="/s/mutation_mapper" getComponent={lazyLoadComponent(MutationMapperTool)} />
```
Comment (onursumer) [inline src/routes.jsx]: Do we still need to keep the original path `/mutation_mapper`? I think the only place where we refer this route is in `mutation_mapper.jsp` which we can retire now.

File: end-to-end-tests/specs/mutationMapperTool.spec.js (hunk 1 above)
Comment (adamabeshouse) [reply end-to-end-tests/specs/mutationMapperTool.spec.js]: is there a typo CBIOPORTAL_*x*URL

Comment (onursumer) [reply end-to-end-tests/specs/mutationMapperTool.spec.js]: oops, yes :)

**Pull request merged**